modules/packages/DistributedBag.chpl

```diff
@@ -253,7 +254,20 @@ module DistributedBag {
       }
       return chpl_getPrivatizedCopy(unmanaged DistributedBagImpl(eltType), _pid);
     }
-
+  
+    // printing the contents of DistBag onto a channel
```

Comment:
Let's make this a docstring:


```suggestion
    /* Read/write the contents of DistBag from/to a channel */
```

but also lets make sure this method doesn't show up in the documentation

```chpl
   pragma "no doc"
    /* Read/write the contents of DistBag from/to a channel */
   proc readWriteThis(ch: channel) throws {
```